### ISSUE TITLE
New notes aren't shown on homepage straightaway when a new note is added.

### ISSUE THREAD

**Issue by MatthewSloyan** [MatthewSloyan/IT-Professional-Skills-Group-Project#11]

**Comment by MatthewSloyan** [comment]
When updating and deleting notes this issue persists.

**Comment by MatthewSloyan** [comment]
I tried to fix this by creating a method that would reload the list, which I would then call at the end of the add/delete methods but it still doesn't seem to fix the issue.

**Comment by MatthewSloyan** [comment]
I have finally implemented a fix for this issue. After I implemented the pull to refresh functionality from the ionic docs, I found that there is a setTimeout() method which waits a specific amount of time and then calls the method inside the brackets. I thought the issue could be the fact that the reload list was being called too quickly before the database got a chance to update, so essentially it was reloading the same list. To fix this I have implemented the setTimeout() with a half a second delay in each of the add/delete/update methods (Seen below).  

This improves user experience drastically as now the user sees their real-time updates, deletes and additions.

![image](https://user-images.githubusercontent.com/26766108/56151442-27010200-5fa9-11e9-8136-2116e7515b46.png)
